ophyd/status.py

```diff
@@ -569,6 +569,16 @@ def __str__(self):
             "".format(self.__class__.__name__, self)
         )
 
+    def __contains__(self, status: StatusBase):
```

Comment:
Nit: use `bluesky.protocols.Status` rather than `StatusBase`, provides a little more flexibility.

Reply:
Does this work? Seems to fail the tests as it looks like we have no current dependency on `bluesky`?

Reply:
Ah, I didn't realise the bluesky dependency is dev-only, my bad! Can either ignore this comment and keep `StatusBase` or couch the import in `if TYPE_CHECKING`, I don't mind

Reply:
Given the lack of use of `bluesky.protocols.Status` generally in v1 code I will revert to `StatusBase`